### PR TITLE
DS-196 Generate expected i18n keys list

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "build": "vite build",
     "build-tokens": "node design-tokens/build.js",
     "build-storybook": "vite build & storybook build",
-    "intl": "formatjs extract src/components/**/*.ts*",
+    "intl-keys": "formatjs extract src/components/**/*.ts* --out-file i18n/keys.json",
```
Comment (gregaubert) [inline package.json:26]: Should we make it part of the build command?

**Review by gregaubert — APPROVED** [review]
Nice! :) I actually didn't know we had a command already.
I guess it would also make sense to document a link to that file in the readme where we talk about the react-intl provider.

File: package.json (hunk 1 above)
Comment (gregaubert) [reply package.json:26]: Can be another ticket then

Comment (jeremy-davis-sonarsource) [reply package.json:26]: I considered it, and I think the `releasability check` is better

**Pull request merged**